package.json

```diff
@@ -8,7 +8,7 @@
     "watch": "mocha --watch $(find test -name '*.test.js')",
     "coverage": "istanbul cover ./node_modules/.bin/_mocha -- $(find test -name '*.test.js')",
     "test-travis": "istanbul cover node_modules/.bin/_mocha --report lcovonly -- $(find test -name '*.test.js')",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release"
```

Comment:
Can you explain this change pls?
Never used semantic release, so i have no idea.

Reply:
Me neither 😉 

It's part of the breaking changes that came with version 9.0:

https://github.com/semantic-release/semantic-release/releases/tag/v9.0.0

Reply:
Alright 😬